R/path.R

```diff
@@ -1,17 +1,15 @@
 #' Path computations
 #'
-#' All functions apart from [path_expand()] and [path_real()] are purely
```

Comment:
`path_expand()` is not actually documented in this file, so I reworded

Reply:
Ah, I think this was changed later, thanks!